src/widget.js

```diff
@@ -86,6 +86,11 @@ export default class Widget extends Plugin {
 		const viewDocument = editor.editing.view;
 		let element = domEventData.target;
 
+		// Widget is not selectable when editor is read-only.
```

Comment:
Why widget is not selectable when editor is read-only? I mean: what's wrong in selecting the widget when the editor is in the read-only mode?